ConfCore/TranscriptIndexer.swift

```diff
@@ -29,7 +29,7 @@ public final class TranscriptIndexer {
         self.manifestURL = manifestURL
     }
 
-    fileprivate let queue = DispatchQueue.global(qos: .utility)
+    fileprivate let queue = DispatchQueue(label: "Transcript Indexer", qos: .background)
```

Comment:
Realm only works with serial queues.

Reply:
True. Using `.global` is usually not a good idea anyways 😬